docs/Project-Configuration/Project-Commands.md

```diff
@@ -1202,6 +1203,7 @@ Parameters:
     `local_file`
 -   `remote_file`: the S3 path to get the file from
 -   `bucket`: the S3 bucket to use.
+-   `region`: AWS region of the bucket, defaults to us-east-1.
```

Comment:
This parameters documentation has to be added to s3.put as well.

Reply:
S3.put actually already has this documented, I just thought it'd be nice to put it in the example as well